addons/digest/models/digest.py

```diff
@@ -112,8 +113,12 @@ def action_deactivate(self):
     def action_set_periodicity(self, periodicity):
         self.periodicity = periodicity
 
-    def action_send(self):
-        to_slowdown = self._check_daily_logs()
+    def action_send(self, manual=False):
+        """ Send digest emails, compute next send date and unless manual, update periodicity based on recipients being
```

Comment:
now the first line is redundant with the param section, maybe just
```
Send digests email to all the registered users
```

(or something like that)